### PR TITLE
Further reduce logging

### PR DESCRIPTION
**Pull request by sopel39**
FYI: @electrum

### CODE REVIEW

File: presto-main/etc/log.properties [hunk 1]
```diff
@@ -9,3 +9,4 @@ io.prestosql=INFO
 com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory=WARN
 com.ning.http.client=WARN
 io.prestosql.server.PluginManager=DEBUG
+org.apache.parquet.hadoop=WARN
```
Comment (kokosing) [inline presto-main/etc/log.properties]: is this file used during tests?

I would log all logs `org.apache` at WARN level

File: presto-hive/src/test/java/io/prestosql/plugin/hive/HiveQueryRunner.java [hunk 2]
```diff
@@ -126,6 +128,12 @@ public static DistributedQueryRunner createQueryRunner(Iterable<TpchTable<?>> ta
         }
     }
 
+    private static void setupLogging()
+    {
+        Logging logging = Logging.initialize();
+        logging.setLevel("org.apache.parquet.hadoop", WARN);
```
Comment (kokosing) [inline presto-hive/src/test/java/io/prestosql/plugin/hive/HiveQueryRunner.java]: I would setup this to `org.apache`, then I would move it to `DistributedQueryRunner`

Comment (sopel39) [reply presto-hive/src/test/java/io/prestosql/plugin/hive/HiveQueryRunner.java]: There could be useful INFO `org.apache` logs though

File: presto-main/etc/log.properties (hunk 1 above)
Comment (electrum) [reply presto-main/etc/log.properties]: It is only used if you run the server manually in the IDE.

Comment (electrum) [reply presto-main/etc/log.properties]: Do product tests use these somehow?

Comment (sopel39) [reply presto-main/etc/log.properties]: I'm not sure TBH

Comment (sopel39) [reply presto-main/etc/log.properties]: This is actually used by `HIVE_TESTS=true` too (so `TestHiveClient` integration tests): https://travis-ci.com/prestosql/presto/jobs/173791261
but those tests are not problematic.

**Pull request merged**